src/language/syntax/rules/profile/profile.ts

```diff
@@ -88,7 +88,8 @@ export const PRIMITIVE_TYPE_NAME: SyntaxRule<
 export const ENUM_VALUE: SyntaxRule<WithLocation<EnumValueNode>> =
   documentedNode(
     expectTerminated(
-      SyntaxRule.identifier().followedBy(
+      SyntaxRule.followedBy(
+        SyntaxRule.identifier(),
```

Comment:
I do not understand this change

Reply:
I've finally figured out how to correctly type variadic generic type transforms, so I went from a member function `rule.followedBy(anotherRule)` to `SyntaxRule.followedBy(rule, anotherRule, ...moreRules)`

Reply:
Seems you know what you are doing :)